default.nix

```diff
@@ -1,8 +1,34 @@
 { fly-base ? (import ./fly-base.nix {})
 }:
 (fly-base.fly.evalSpec) {
-  config = {
+  config = { pkgs, ... }: {
```

Comment:
This is needed to get a reference to `pkgs`  for `additionalLayers`.

This works because `config` is a module, and modules are either a simple attrset *or* a function returning an attrset, function accepting a few helpful inputs like `lib, config, pkgs`.